platform/logging/appenders/base/BaseAppender.ts

```diff
@@ -0,0 +1,24 @@
+import { LogRecord } from '../../LogRecord';
```

Comment:
There is a chance that I get rid of additional folders for appenders and it's config. But it seems that `rolling-file-appender` will require a lot of additional code (roll over policy handling) and having everything in one file can be a mess...